newscoop/src/Newscoop/PluginGeneratorBundle/Resources/skeleton/bundle/composer.json.twig

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "newscoop/{{ pluginName | lower }}-plugin-bundle",
```

Comment:
ah, yeah, good point.  I can add a 'vender' input option to the generator command and go through and find out how many places that will need to be updated.  In plugins dir that would mean the path would be:
  plugins/{vender}/{pluginName} 

namespace would then be:
  {vender}/{pluginName}PluginBundle

and bundle:
  {vender}{pluginName}PluginBundle

does that sound right?